install/project/includes/configuration/configuration.inc.sample.php

```diff
@@ -165,9 +166,11 @@
 			define ('__APP_CSS_ASSETS__', __PROJECT_ASSETS__ . '/css');
 			define ('__APP_IMAGE_ASSETS__', __PROJECT_ASSETS__ . '/images');
 			define ('__APP_PHP_ASSETS__', __PROJECT_ASSETS__ . '/php');
+			define ('__APP_IMAGE_CACHE__', __APP_IMAGE_ASSETS__ . '/cache');
+			define ('__APP_UPLOAD__', __DOCROOT__ . __PROJECT_ASSETS__ . '/upload');
```

Comment:
We must then distinguish between that which the assets or the temporary folder (TMP). I do not like a big mess, this is too much and cache folders everywhere. This is not good. Here you can agree on that, whether it is sufficient for the development or production of, for example, from two places:
- project/assets (permanent stuff)
  - uploads
  - images
  - cache
  - i18n.po
- project/tmp (temporary things that could, if necessary, to put Cron periodic cleaning, etc.)
  - purifier
  - soap
  - etc

It is easier to track and maintain so. What do your thinks?